Frameworks/QuartzCore/CALayer.mm

```diff
@@ -2477,24 +2484,31 @@ + (CGRect)convertRect:(CGRect)pos fromLayer:(CALayer*)fromLayer toLayer:(CALayer
         return pos;
     }
 
-    if (!fromLayer) {
-        // get the top most layer for the current layer hierarchy
+    UIElement fromLayerElement = nullptr;
+    UIElement toLayerElement = nullptr;
+
+    if (fromLayer) {
+        // Get the backing sublayer xaml UIElement for fromLayer
+        // We use the sublayer element to support proper point conversion within a scrolled view.
+        fromLayerElement = [fromLayer _getSublayerXamlElement];
+    } else if (GetCACompositor()->IsRunningAsFramework()) {
+        // In framework mode we don't own the topmost layer
+        fromLayerElement = Window::Current().Content();
```

Comment:
I think so, yes.

---
In reply to: [113999848](https://github.com/Microsoft/WinObjC/pull/2650#discussion_r113999848) [](ancestors = 113999848)